app/models/goal.py

```diff
@@ -3,4 +3,11 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal")
+    
+    
+
+    # def to_string(self):
+    #     return f"{self.id}: {self.title} {self.description} {self.completed_at}"
```

Comment:
I was going to mention adding an instance method here. Here is a suggestion on how it could look
```suggestion
    def to_json(self):
        return {
            "id": self.goal_id,
            "title": self.title,
        }
```